public/app/features/trash-section/TrashPage.tsx

```diff
@@ -1,15 +1,85 @@
-import React from 'react';
+import React, { memo, useEffect } from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
 
+import { FilterInput } from '@grafana/ui';
 import { Page } from 'app/core/components/Page/Page';
+import { t } from 'app/core/internationalization';
+import { ActionRow } from 'app/features/search/page/components/ActionRow';
+import { getGrafanaSearcher } from 'app/features/search/service';
+
+import { useDispatch } from '../../types';
+import { Props as DashboardRouteComponentsProps } from '../browse-dashboards/BrowseDashboardsPage';
+import { BrowseView } from '../browse-dashboards/components/BrowseView';
+import { SearchView } from '../browse-dashboards/components/SearchView';
+import { getFolderPermissions } from '../browse-dashboards/permissions';
+import { setAllSelection } from '../browse-dashboards/state';
+
+import { useTrashStateManager } from './useTrashStateManager';
+
+const TrashPage = memo(({ match }: DashboardRouteComponentsProps) => {
```

Comment:
You need to update this, right?

Reply:
Yes, I just merged the renaming PR and now there are conflicts in this one.